config/locales/doc_auth/en.yml

```diff
@@ -57,7 +57,12 @@ en:
           in brighter lighting.
         visible_photo_check: We couldn't verify the photo on your ID. Try taking new
           pictures.
-      not_a_file: The selection was not a valid file.
+      no_camera:
+        explanation: To continue, sign in to your login.gov account on any device
+          that has a camera, like a mobile phone, tablet or computer with a webcam.
+        heading: You need a camera to verify your identity
+        title: Camera needed
+      not_a_file: The selection was not a valid file
```

Comment:
Was it intended to remove the period from this text?

Reply:
Damned. I missed that. It was a weird conflict that I though git shouldn't have had a problem with.  I didn't see the period. Good catch. I'll fix in a separate PR.